client/src/components/PrivateRoomList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const PrivateRoomList = () => {
+  return(
+    <div>
+      <table className= 'room availability table'>
```

Comment:
For `className`, did you mean to put `room` `availability` `table` all separately? `className` words can be connected with `-`